deployment/utility/sftp_copy.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+from ..clusterObjectModel import cluster_object_model
+from ..k8sPaiLibrary.maintainlib import common
+import logging
+import logging.config
+
+
+class OpenPaiSftpCopy:
+
+    def __init__(self, filename, source, dest, cluster_config_path, filter):
+        self.filename = filename
+        self.source = source
+        self.dest = dest
+        objectModelFactoryHandler = cluster_object_model.cluster_object_model(configuration_path=cluster_config_path)
```

Comment:
What about do not pass the whole object_model? 
For example, here you may only want a `machine-list`.
So we can narrow down which is need by the function.

Reply:
This should be an optimization of cluster object model. And maybe you could fire an new issue to track this.

Reply:
I mean, the function don't need to accept a 'cluster_object_model'.

It's just a pseudo:

machine_list = ...  # how to generate machine list is out of the scope of the class OpenPaiSftp
openPaiSftp = OpenPaiSftp(machine_list) # remove copy from the class name. Class name should better be a `noun`.
result = openPaiSftp.copy(from_dir, to_dir) # So we don't need to call `run`, we call the `verb` copy.

Reply:
Thank you for your suggestion. And the machine-list part will be optimized. But I will keep the naming style as before.